Frontend/ui-driver/src/Screens/ReferralScreen/View.purs

```diff
@@ -32,7 +32,7 @@ import Font.Style as FontStyle
 import JBridge (openUrlInApp, startTimerWithTime, toast)
 import Language.Strings (getString)
 import Language.Types (STR(..))
-import Prelude (Unit, bind, const, pure, unit, ($), (<<<), (==), (<>), map, discard, show, (>), void, (/=), (/), (*), (+), not, (||), negate, (<=), (&&), (-))
+import Prelude (Unit, bind, const, pure, unit, ($), (>=), (<<<), (==), (<>), map, discard, show, (>), void, (/=), (/), (*), (+), not, (||), negate, (<=), (&&), (-), (<))
```

Comment:
@Anuragini-2506 can we please remove >= this operator if this is not being used

Reply:
done